mobly/snippet/client_base.py

```diff
@@ -127,55 +116,42 @@ def start_server(self):
       errors.ServerStartError: when failed to start the snippet server.
     """
 
-    @contextlib.contextmanager
-    def _execute_one_stage(stage):
-      """Context manager for executing one stage.
-
-      Args:
-        stage: StartServerStages, the stage which is running under this
-          context manager.
-
-      Yields:
-        None.
-      """
-      self.log.debug('[START_SERVER] Running the stage %s.', stage.name)
-      yield
-      self.log.debug('[START_SERVER] Finished the stage %s.', stage.name)
-
-    self.log.debug('Starting the server.')
+    self.log.debug('Initializing the snippet package %s.', self.package)
     start_time = time.perf_counter()
 
-    with _execute_one_stage(StartServerStages.BEFORE_STARTING_SERVER):
-      self.before_starting_server()
+    self.log.debug('Preparing to start the snippet server for %s.',
```

Comment:
let's user either "of" or "in" instead of "for" so it's clear the server is part of the package?

or maybe we can use the term "snippet server [com.google.blah.blah]"?

Reply:
Changed the "for" to "of".